### DISCUSSION TITLE
书籍翻译的时候可以自定义颜色吗？

### DISCUSSION THREAD

**Discussion by Apocalypse12345** [immersive-translate/immersive-translate#2035] [Help]
### 概述 | Summary

![bug](https://github.com/user-attachments/assets/78493352-0b58-4f49-aa29-4bb5c59a0ff5)
书籍翻译底部是白色的，翻译出来变成白色，颜色改不了，这样没法看

**Comment by keyboard3** [comment]
你是的夜间主题修改了字体颜色导致的。换其他主题插件。或者关闭它
